cli/app.go

```diff
@@ -1148,9 +1148,33 @@ var app = &cli.App{
 							Action: RobotsPartShellAction,
 						},
 						{
-							Name:        "cp",
-							Usage:       "copy files to and from a machine part",
-							Description: `In order to use the cp command, the machine must have a valid shell type service.`,
+							Name:  "cp",
+							Usage: "copy files to and from a machine part",
+							//nolint:lll
+							Description: `
+In order to use the cp command, the machine must have a valid shell type service.
+Specifying ~ or a blank destination for the machine will use the home directory of the user
+that is running the process (this may sometimes be root).
+Note: There is no progress meter while copying is progress.
+
+Copy a single file to the machine with a new name:
+'viam machine part cp --organization "org" --location "location" --machine "m1" --part "m1-main" my_file machine:/home/user/'
+
+Recursively copy a directory to the machine with the same name:
+'viam machine part cp --organization "org" --location "location" --machine "m1" --part "m1-main" -r my_dir machine:/home/user/'
+
+Copy multiple files to the machine with recursion and keep original permissions and metadata:
+'viam machine part cp --organization "org" --location "location" --machine "m1" --part "m1-main" -r -p my_dir my_file machine:/home/user/some/existing/dir/'
+
+Copy a single file from the machine to a local destination:
+'viam machine part cp --organization "org" --location "location" --machine "m1" --part "m1-main" machine:my_file ~/Downloads/'
+
+Recursively copy a directory from the machine to a local destination with the same name:
+'viam machine part cp --organization "org" --location "location" --machine "m1" --part "m1-main" -r machine:my_dir ~/Downloads/'
+
+Copy multiple files from the machine to a local destination with recursion and keep original permissions and metadata:
+'viam machine part cp --organization "org" --location "location" --machine "m1" --part "m1-main" -r -p machine:my_dir machine:my_file ~/some/existing/dir/'
```

Comment:
maybe one or two of these examples can show using the machine-id directly and omitting org and location.

Reply:
good idea

Reply:
added some language around it and only put the flags in the first example